### DISCUSSION TITLE
latest mmdetection (version 3.3) is not compatible with latest mmcv (latest version 2.2.0)

### DISCUSSION THREAD

**Discussion by tanguanghua** [open-mmlab/mmdetection#11754] [Q&A]
latest mmdetection (version 3.3) is not compatible with latest mmcv (latest version 2.2.0)

**Comment by vchigrinsky** [comment]
perhaps it's worth replacing < with <= here?
https://github.com/open-mmlab/mmdetection/blob/main/mmdet/__init__.py#L17
